app/src/main/java/org/xbmc/kore/host/actions/GetPlaylist.java

```diff
@@ -170,5 +170,26 @@ private GetPlaylistResult(int playlistId, String type, List<ListType.ItemsAll> i
             this.type = type;
             this.items = items;
         }
+
+        @Override
+        public boolean equals(Object obj) {
+            if (obj == this)
+                return true;
+
+            if ((! (obj instanceof GetPlaylistResult)) )
+                return false;
+
+            GetPlaylistResult other = (GetPlaylistResult) obj;
+
+            if (id != other.id || ! type.equals(other.type))
+                return false;
+
+            for(int i = 0; i < items.size(); i++) {
+                if (items.get(i).equals(other.items.get(i)))
```

Comment:
Can you please check if this is right. Seems to be missing a !. 
It's also assuming that the items are in the same order, which should be validated.

Reply:
Thx, negation added.
If I'm correct playlists with the same items but in a different order should be considered different to handle the situation when the user reordered the playlist items on Kodi.